src/components/Modal/Modal.scss

```
@@ -76,4 +76,10 @@ $modal-transform-transition: transform $modal-transition-duration ease-out;
     &[data-floating-ui-status='close'] #{$block}__content {
         transition: $modal-height-transition, $modal-transform-transition;
     }
```

Can we use same selectors block, to reduce selectors duplication?

```suggestion
    &[data-floating-ui-status='close'] #{$block}__content {
        transition: $modal-height-transition, $modal-transform-transition;
        
        @media (prefers-reduced-motion: reduce) {
            transition: none;
        }
    }
```

---

Then this code must be repeated 4 times